instrumentation/kafka/kafka-clients/kafka-clients-2.6/library/README.md

```diff
@@ -72,12 +72,32 @@ Then use the `tracingConsumer` as usual for receiving messages from the Kafka cl
 ### Usage (Metrics)
 
 The Kafka client exposes metrics via `org.apache.kafka.common.metrics.MetricsReporter` interface.
-OpenTelemetry provides an implementation that bridges the metrics into OpenTelemetry.
+OpenTelemetry provides an implementation that bridges the metrics into OpenTelemetry. There are two
+options for capturing metrics, manually instrumenting or configuring JMX Metrics, both described below.
+
+#### Manual instrumentation
 
 To use, merge the config properties
 from `KafkaTelemetry.create(OpenTelemetry).metricConfigProperties()`
 with the configuration used when creating your producer or consumer.
 
+#### Configuring JMX Metrics
+
+the `kafka.consumer` and `kafka.producer` JMX Metrics can be configured as described
+in [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics/javaagent#configuration-files),
```

Comment:
we use relative paths elsewhere to link within this project

Reply:
@laurit I was hesitant to create a new doc in case this fit in here. As you've mentioned below it's to do with JMX not manual instrumentation. Where do you think this document should live? Perhaps in https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/kafka/kafka-clients/kafka-clients-0.11? Or append to the existing JMX configuration files at https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics/javaagent#configuration-files as suggested below?

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/jmx-metrics/javaagent seems like a better fit because that is the instrumentation you are really using when gathering metrics via jmx. I'd just follow how the existing rules are described.

Reply:
@laurit let me know what you think ^ 😄

Reply:
I think as they are the jmx rules you have added do not align well with the existing rules. As you noted the existing metrics use `.` as separator, don't have a `jmx` prefix, define units, description etc. Existing rules are listed in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/README.md#predefined-metrics that links to a page that lists the metrics. Existing metrics rules are tested in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jmx-metrics/javaagent/src/test/java/io/opentelemetry/instrumentation/javaagent/jmx/JmxMetricInsightInstallerTest.java

Reply:
@laurit I have just upgraded the javaAgent in my project to the latest version `1.31.0` from `1.30.0`, and Kafka Client metrics have magically started appearing without the need for manual configuration. I have had a dig through the [commits](https://github.com/open-telemetry/opentelemetry-java-instrumentation/compare/v1.30.0...v1.31.0) but can't see anything that would point to this suddenly working. Do you perhaps know if something has changed since the last version that would cause the auto-instrumentation of metrics to start working for Kafka?


Here are some PRs since `1.30.0` that reference the Kafka metrics
- https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9654
- https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9619
- https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/9508

I'm unfamiliar with the internals so not sure if any of these caused the fix.

Reply:
@bcarter97 Sorry I wasn't able to spot any changes between `1.30.0` and `1.31.0` that could affect kafka metrics.

Reply:
I will leave this as just a document cleanup for the missing metric descriptions if that's alright.